### PR TITLE
[Nightly] Add Binance chains

### PR DESCRIPTION
**Pull request by joaquim-verges**
@nachoiacovino @jnsdls need to add binance to the chain map here, then add this build to embeds

### CODE REVIEW

File: packages/thirdweb-react/src/constants/chain.ts [hunk 1]
```diff
@@ -185,6 +185,32 @@ const chain: Record<string, Chain> = {
     ],
     testnet: true,
   },
+  binanceSmartChainMainnet: {
+    id: ChainId.BinanceSmartChainMainnet,
+    name: "Binance Smart Chain Mainnet",
```
Comment (jnsdls) [inline packages/thirdweb-react/src/constants/chain.ts:190]: can we make this `BSC Mainnet`

File: packages/thirdweb-react/src/constants/chain.ts [hunk 2]
```diff
@@ -185,6 +185,32 @@ const chain: Record<string, Chain> = {
...
+      {
+        name: "BscScan",
+        url: "https://bscscan.com/",
+      },
+    ],
+    testnet: false,
+  },
+  binanceSmartChainTestnet: {
+    id: ChainId.BinanceSmartChainTestnet,
+    name: "Binance Smart Chain Testnet",
```
Comment (jnsdls) [inline packages/thirdweb-react/src/constants/chain.ts:203]: `BSC Testnet`?

**Review by jnsdls — COMMENTED** [review]
the name is just so long other than that LGTM

File: packages/thirdweb-react/src/constants/chain.ts (hunk 1 above)
Comment (jnsdls) [inline packages/thirdweb-react/src/constants/chain.ts:190]:
```suggestion
    name: "BSC Mainnet",
```

File: packages/thirdweb-react/src/constants/chain.ts (hunk 2 above)
Comment (jnsdls) [inline packages/thirdweb-react/src/constants/chain.ts:203]:
```suggestion
    name: "BSC Testnet",
```

**Review by jnsdls — APPROVED** [review]
update chain names and then ship

**Comment by nachoiacovino** [comment]
@jnsdls if we don't have the names we had MetaMask is gonna give a warning when adding them, just a heads up

**Comment by jnsdls** [comment]
> @jnsdls if we don't have the names we had MetaMask is gonna give a warning when adding them, just a heads up

okay in that case we need to have 2 name fields then, 1 for the chain and one that we'll show in the network selector

**Comment by jnsdls** [comment]
will be merged in: https://github.com/thirdweb-dev/js/pull/12

**Pull request closed without merging**